parser/shared/src/main/scala/hmda/parser/fi/lar/LarCsvParser.scala

```diff
@@ -0,0 +1,98 @@
+package hmda.parser.fi.lar
+
+import hmda.model.fi.lar._
+
+object LarCsvParser {
+  def apply(s: String): LoanApplicationRegister = {
+    val values = s.split('|').map(_.trim)
```

Comment:
In the long run, I think we'll want a real CSV (or PSV in our case) parsing library.  I'm guessing there _will_ be legit reasons for `|` characters, which would cause the string to be split incorrectly.